tools/templates/profile.mustache

```diff
@@ -128,9 +128,11 @@
      {{#user}}
      options.user = {
        email: '{{user.email}}',
-       iss: '{{user.claims.iss}}',
```

Comment:
Can we keep the `iss` param here? I think it is important to note **who** this is authorized for.

Reply:
I'm not sure if we still get the "issuer" property in the user object.
We do have the "subject" property which is being displayed but the "issuer" claim is only available in the id_token object which we don't have access to in our backend anymore.

Reply:
As we discussed yesterday, it might make sense to iterate over the dictionary and pop the `key`, `value` pairs.

See: https://stackoverflow.com/a/17516837/7059401

Reply:
As already mentioned, these templates are going to be over-written by the @okta/samples generator once https://github.com/okta/samples-js-angular-1/pull/24/files is merged